administrator/components/com_associations/View/Association/HtmlView.php

```diff
@@ -169,6 +171,27 @@ public function display($tpl = null)
 			$this->targetTitle      = AssociationsHelper::getTypeFieldName($extensionName, $typeName, 'title');
 			$task                   = $typeName . '.' . $this->targetAction;
 
+			// The update layout can only be set when a default association language is used.
+			// Get version ids of the parent, when versions are enabled.
```

Comment:
https://developer.joomla.org/coding-standards/inline-code-comments.html
```suggestion
			/** 
			 * The update layout can only be set when a default association language is used.
			 * Get version ids of the parent, when versions are enabled.
			 */
```